telegram_dispatch.js

```diff
@@ -0,0 +1,73 @@
+const request = require('request');
+const express = require('express');
+const app = express();
+const TelegramBot = require('node-telegram-bot-api');
+const fs = require('fs');
+
+// const aws = require('aws-sdk');
+
+// let s3 = new aws.S3({
+//   accessKeyId: process.env.AWS_KEY,
+//   secretAccessKey: process.env.AWS_SECRET
+// });
+
+const sqs = require('sqs');
+
+const queue = sqs({
+   access: process.env.AWS_KEY,
+   secret: process.env.AWS_SECRET,
+   region: 'us-east-1' // defaults to us-east-1
+});
+const production_queue_name = "intelligenttrading-sqs-production"
+
+const token = process.env.TELEGRAM_BOT_TOKEN;
+const bot = new TelegramBot(token, { polling: false });
+const telegram_message_options = {
+  parse_mode: "Markdown"
+};
+
+
+function onSQSMessage(message_data, callback) {
+  if (message_data.type == "trade signal"){
+    var risk = message_data.risk
+
+    request(`https://${process.env.ITT_API_HOST}/users?risk=${risk}`, function (error, response, body) {
+        if (!error && response.statusCode == 200) {
+            var data = JSON.parse(body)
+            data['chat_ids'].foreach(function(chat_id){
+              bot.sendMessage(chat_id, ("this is a trade signal"));
```

Comment:
The message template should be 
```
BTC/USD    
Bullish 🔵⚪️⚪️    
Near horizon, Bittrex    
Price: $4814.0 (-0.28%)
```
and therefore message_data should be
```
coin=BTC
market=Poloniex
signal=SMA ( | trade???)
trend=BEARISH
strength=(1,3)
price=4814
price_change=0.0028
```

I think `message_data.type` is `message_data.signal` with the value == "SMA"